File: src/main/java/com/jinkyumpark/bookitout/search/apiResponse/aladin/ApiAladinUsedList.java
package com.jinkyumpark.bookitout.search.apiResponse.aladin;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;

@NoArgsConstructor @AllArgsConstructor
@Getter
public class ApiAladinUsedList {
    private ApiAladinUsed aladinUsed;
    private ApiAladinUserUsed userUsed;
    private ApiAladinSpaceUsed spaceUsed;
}

File: search/src/main/kotlin/com/jinkyumpark/search/used/apiResponse/aladin/ApiAladinUsedList.kt
package com.jinkyumpark.search.used.apiResponse.aladin

data class ApiAladinUsedList(
    val aladinUsed: ApiAladinUsed,
    val userUsed: ApiAladinUserUsed,
    val spaceUsed: ApiAladinSpaceUsed,
)
